sdk/daml-lf/transaction/src/main/scala/com/daml/lf/Error.scala

```diff
@@ -171,8 +171,11 @@ object Error {
 
     // TODO https://github.com/digital-asset/daml/issues/17647:
     //  - add coid, srcTmplId (alternatively pkgId of srcTmplId), and dstTempId
```

Comment:
Adding in this source based information is considered to be involved, and so is not planned to be done within the current iteration of UP-20